src/woo_publications/logging/logevent.py

```diff
@@ -0,0 +1,166 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from django.db import models
+    from ..accounts.models import User
+
+from .constants import Events
+
+__all__ = [
+    # admin
+    "audit_admin_create",
+    "audit_admin_read",
+    "audit_admin_update",
+    "audit_admin_delete",
+    # api
+    "audit_api_create",
+    "audit_api_read",
+    "audit_api_update",
+    "audit_api_delete",
+]
+
+"""must contain either a django_user or user_id + user_display"""
+
+
+def _audit_event(
+    content_object: models.Model,
+    event: Events,
+    user_id: str | None = None,
+    user_display: str | None = None,
+    django_user: User | None = None,
+    **kwargs: any,
```

Comment:
Either use `from typing import Any` or leave it untyped, I prefer the latter

`any` is not a type, it's a builtin function, `typing.Any` is a type though